src/main/java/org/xstefank/check/TemplateChecker.java

```diff
@@ -31,22 +31,23 @@ public TemplateChecker() {
     }
 
     public void checkPR(JsonNode payload) {
-        StringJoiner joiner = new StringJoiner(", ");
+        //StringJoiner joiner = new StringJoiner(", ");
+        String description = "";
         for (Check check : checks) {
             String message = check.check(payload);
             if (message != null) {
-                joiner.add(message);
+                description = message;
             }
         }
 
         log.info("updating status");
-        String description = joiner.toString();
+        //String description = joiner.toString();
```

Comment:
Please do not commit commented code. In version control we can see what code was there before :).